src/Renderer/TableCell.php

```diff
@@ -62,12 +62,9 @@ function render(Frame $frame)
             $this->_render_outline($frame, $border_box);
         }
 
-        $id = $frame->get_node()->getAttribute("id");
-        if (strlen($id) > 0) {
-            $this->_canvas->add_named_dest($id);
-        }
-
-        // $this->debugBlockLayout($frame, "red", false);
+        $this->addNamedDest($node);
+        $this->addHyperlink($node, $border_box);
```

Comment:
To handle a scenario where the anchor element is styled `display: table-cell`?

Reply:
Yes, a bit of an edge case, but it works in browsers, too.  Just noticed that it makes sense on table rows and row groups, as well.

Reply:
The things you can do...